src/routes/(admin)/tests/test-[type]/[id]/question_table/data-table.svelte

```
@@ -192,23 +197,18 @@
-				});
-				$formData.question_revision_ids = tempArray;
 				open = false;
 			}}
 		>
-			Add to Test {$formData.is_template ? ' Template' : ''}
+			Add to Test {isTemplate ? ' Template' : ''}
 		</Button>
 		<div class="my-auto ml-10 flex-1 text-sm font-normal">
-			{table.getFilteredSelectedRowModel().rows.length} of{' '}
-			{table.getFilteredRowModel().rows.length} question(s) selected.
+			{Object.entries(rowSelection).length} of{' '}
```

ESLint

[error] Unexpected mustache interpolation with a string literal value.

(svelte/no-useless-mustaches)